src/Types.ts

```diff
@@ -52,3 +52,18 @@ type RegionInfo = {
   // Numerical string
   start: string[]
 }
+
+// Available color sets
+enum examplecolors{
+  greys,
+  ygreys,
+  blues,
+  reds,
+  plainColors,
+  lightColors
+}
+
+// tubemap.js
+// Stores the assigned color of the track system, also stores boolean colorReadsByMappingQuality
+// Each entry contains the coloring of 1 haplotype file or 1 read file
+type colors = [{"forward": examplecolors, "reverse": examplecolors, "colorReadsByMappingQuality": boolean}]
```

Comment:
It would also be good to explain here how this manages to represent coloring for *either* a haplotype/path set *or* a set of reads; it looks like this is managed by just always using the "forward" field for haplotypes and paths and ignoring the others. That's an *okay* solution, but it is going to be confusing if we always have to say `color_scheme.forward` for haplotypes when forward-ness has nothing to do with what we're doing. We also seem to still always set the reverse scheme to some real color scheme which makes it hard to tell it isn't always used.

Maybe the fields could be renamed to generalize more neatly? We could have something like `mainPalette` for forward, and then `auxPalette` which would be either a reverse strand color scheme for reads or `undefined` for haplotypes. And the one `colorReadsByMappingQuality` field which is always `false` and meaningless for haplotypes/paths could be something like `colorDataSource` or `displayedOverlay` or `colorBy` which could be `mapping_quality` or unset for reads, and could be always unset for paths/haplotypes. Or it could be `colorByMappingQuality` and true or false for reads and `undefined` for paths/haplotypes.